internal/buildscripts/packaging/tests/package_test.py

```diff
@@ -62,6 +64,23 @@ def get_libcap_command(container):
     else:
         return "zypper install -y libcap-progs"
 
+@pytest.mark.parametrize(
+    "distro",
+    [pytest.param(distro, marks=pytest.mark.tar) for distro in TAR_DISTROS]
+)
+def test_tar_collector_package_install(distro):
+    pkg_path = get_package(distro, PKG_NAME, PKG_DIR)
+    assert pkg_path, f"{PKG_NAME} package not found in {PKG_DIR}"
+    pkg_base = os.path.basename(pkg_path)
+
+    with run_distro_container(distro) as container:
+
+        copy_file_into_container(container, pkg_path, f"/test/{pkg_base}")
+        run_container_cmd(container, f"tar xzf /test/{pkg_base} -C /tmp")
+        run_container_cmd(container, f"test -d /tmp/bin")
+        run_container_cmd(container, f"test -d /tmp/signalfx-agent")
```

Comment:
To work w/ the SA extension defaults I think it'd be preferable to rename this dir to `agent-bundle` like the other installers do so that users can extract to `/usr/lib/splunk-otel-collector` and run the patch-interpreter script @jeffreyc-splunk mentioned.

https://github.com/signalfx/splunk-otel-collector/blob/main/internal/extension/smartagentextension/factory.go#L59

Reply:
It may be a better experience to add something like a `configure.sh` that does the patch step for users, but that can be in a later PR.

Reply:
OK, I have changed the packaging to follow that direction.

Reply:
Thanks, validated with integration tests.